index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">    <title>Marco's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo"><img src="images/logo.svg"></a>
+        <a href="#" class="hamburger"><img src="images/hamburger.svg"/></a>
+    </header>
+
+    <div class="headline">
+        <h1>Marco Almada</h1>
+    </div>
+    <div class="description">
+        <h2>Hello! I am a software developer! 
+            I can help you build a product, feature or website. 
+            Take a look of my works.
+            If you like what you see and have a project you need coded, 
+            don’t hesitate and contact me.
+        </h2>
+        <button class="collab">Start collaboration</button>
+    </div>
+
```

Comment:
- [ ] great job. Please reduce the gap between the text and start collaboration button like that of Figma.
- [ ] also, you are missing the image under the start collaboration button . please, kindly add that.

yours
![Screenshot (634)_LI](https://user-images.githubusercontent.com/65298048/205687206-ddaf7a7a-dd45-4202-806f-53545e9c3ad1.jpg)
figma
![Screenshot (633)](https://user-images.githubusercontent.com/65298048/205687320-19aa2767-c9ec-4734-87cd-1f9cd129d45e.png)